src/components/header/HeaderComponent.vue

```diff
@@ -291,3 +291,9 @@ const badgeCss = 'fr-badge fr-badge--sm fr-badge--' + props.badgeStyle
   gap: 0.25em;
 }
 </style>
+
+<style>
```

Comment:
```suggestion
<style scoped>
```

On essaie de scoper le style des composants quand c'est possible (d'autant plus quand il y a un `!important`).

Reply:
Je retire cette modification, il y avait un impact au hover qui m'avait échappé